Source/Events.Store.MongoDB/Processing/Streams/Partitioned/StreamProcessorStateExtensions.cs

```diff
@@ -19,16 +19,5 @@ public static class StreamProcessorStateExtensions
         /// <returns>The converted <see cref="Runtime.Events.Processing.Streams.Partitioned.FailingPartitionState" />.</returns>
         public static Runtime.Events.Processing.Streams.Partitioned.FailingPartitionState ToRuntimeRepresentation(this FailingPartitionState state) =>
             new Runtime.Events.Processing.Streams.Partitioned.FailingPartitionState(state.Position, state.RetryTime, state.Reason, state.ProcessingAttempts, state.LastFailed);
```

Comment:
Could some usings make this a bit shorter?

Reply:
Shortened.